src/fdb5/toc/TocMoveVisitor.cc

```diff
@@ -22,8 +22,7 @@
 #include "fdb5/toc/TocMoveVisitor.h"
 #include "fdb5/toc/RootManager.h"
 
-#include <dirent.h>
```

Comment:
dirent is used. This sill works because this header is transitively included by other headers but IMO it should be kept here.